src/index.html

```diff
@@ -22,6 +22,370 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="/src/images/logo.png"
```

Comment:
Avoid using absolute paths for images, use relative paths instead. This ensures that the image can be accessed regardless of the domain or directory structure.